### PR TITLE
Extend to execute integration tests against an external endpoint

### PR DESCRIPTION
**Pull request by sashikamw**

### CODE REVIEW

File: integration-tests/external-endpoint/testplan-props.properties [hunk 1]
```diff
@@ -0,0 +1,15 @@
+OS=LINUX
```
Comment (kasunbg) [inline integration-tests/external-endpoint/testplan-props.properties]: Is it correct to commit the testplan-props.properties content?

File: integration-tests/external-endpoint/const.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+ZIP_FILE_EXTENSION = ".zip"
+CARBON_NAME = "carbon.zip"
+VALUE_TAG = "{http://maven.apache.org/POM/4.0.0}value"
+SURFACE_PLUGIN_ARTIFACT_ID = "maven-surefire-plugin"
```
Comment (kasunbg) [inline integration-tests/external-endpoint/const.py]: unused variable?

File: integration-tests/external-endpoint/infrastructure.properties [hunk 3]
```diff
@@ -0,0 +1,7 @@
+DatabaseHost=localhost
+DatabasePort=3306
+WSO2PublicIP=localhost
+WSO2MgtConsoleURL=https\://localhost\:9443/carbon
+osUser=admin
+DBUsername=root
+DBPassword=123
```
Comment (kasunbg) [inline integration-tests/external-endpoint/infrastructure.properties]: Same as testplan-props.properties.

File: integration-tests/external-endpoint/testplan-props.properties (hunk 1 above)
Comment (sashikamw) [reply integration-tests/external-endpoint/testplan-props.properties]: Removed the properties files and all unused variables

**Pull request merged**